app/Models/Entry.php

```diff
@@ -447,12 +447,14 @@ public function loadCompleteContent($force = false) {
 						$feed->attributes()
 					);
 					if ('' !== $fullContent) {
+						$fullContent = "<!-- FULLCONTENT start //-->{$fullContent}<!-- FULLCONTENT end //-->";
+						$originalContent = preg_replace('#<!-- FULLCONTENT start //-->.*<!-- FULLCONTENT end //-->#s', '', $this->content());
```

Comment:
There might be a risk in some situations that the end of a very large article might be truncated.
I therefore suggest searching for `<!-- FULLCONTENT start //-->`, and getting the rest.
Why is `<!-- FULLCONTENT end //-->` needed?

Reply:
The `<!-- FULLCONTENT end //-->` is needed when you preprend the content. The `<!-- FULLCONTENT start //-->` is needed when you append the content.